### PR TITLE
Add featureOff flag on sidebar of upload page

### PR DESCRIPTION
**Pull request by hmu1540**

### CODE REVIEW

File: src/public/infernode/views/Manage/ManagePage.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export default function ManagePage(): JSX.Element {
 
   return (
     <Stack direction="horizontal" gap={3}>
-      <ManageSidebar />
+      { captureSidebar ? <ManageSidebar /> : <></>}
```
Comment (seadubs) [inline src/public/infernode/views/Manage/ManagePage.tsx:20]: Should we use a new, separate manageSidebar state to flag this off, so we can turn on one sidebar when we've built it without turning on the other?

File: src/public/infernode/components/layout/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ export default function NavBar(): JSX.Element {
               <Nav.Link>Capture</Nav.Link>
             </LinkContainer>
             <LinkContainer to="/manage">
-              <Nav.Link>Manage</Nav.Link>
+              <Nav.Link>Upload</Nav.Link>
```
Comment (seadubs) [inline src/public/infernode/components/layout/NavBar/NavBar.tsx:48]: It looks like `src/__tests__/navbarUnit.test.tsx` needs to be updated with the new navBar page name as well.

**Review by seadubs — CHANGES_REQUESTED** [review]

File: src/public/infernode/components/layout/NavBar/NavBar.tsx (hunk 2 above)
Comment (hmu1540) [reply src/public/infernode/components/layout/NavBar/NavBar.tsx:48]: Gotcha.

**Review by seadubs — APPROVED** [review]
Thanks for the updates, looks great!

**Pull request merged**